packages/build/config/tsconfig.common.json

```diff
@@ -6,10 +6,10 @@
     "noImplicitAny": true,
     "strictNullChecks": true,
 
-    "lib": ["es2017", "dom"],
+    "lib": ["es2018", "dom"],
```

Comment:
This is important - it tells `tsc` to allow new syntax and APIs added by ES2018.